src/hardware/sblaster.cpp

```diff
@@ -2011,32 +2094,41 @@ class SBLASTER final : public Module_base {
 		sb.mixer.enabled=section->Get_bool("sbmixer");
 		sb.mixer.stereo=false;
 
-		Find_Type_And_Opl(section,sb.type,oplmode);
-		configure_filters(section, oplmode);
+		sb.type = find_sbtype();
+		oplmode = find_oplmode();
```

Comment:
find_sbtype() and find_oplmode() --> perfect separation of functionality, reads so much nicer in the functions too. :+1:

Reply:
These are the first untangling steps to separate the SB/OPL/CMS even further into the future.